airflow/providers/amazon/aws/operators/base.py

```diff
@@ -0,0 +1,97 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Just noticed the filename, can you change it to base_aws.py to match the equivalent hook?

Reply:
makes sense